airtable.go

```diff
@@ -40,12 +40,11 @@ func NewTable(name string, account Account) Table {
 	}
 }
 
-// List returns a list of records from the Airtable.
```

Comment:
Any reason this comment was removed? It was there for the sake of generated godocs: https://godoc.org/github.com/mottaquikarim/go-airtable#GenericTable.List

Reply:
This is a weird diff, I did not actually remove this comment, it is still showing up on my branch 🤔

Reply:
Ok cool - nvm then! Resolving convo 👍